Jetcaster/app/src/main/java/com/example/jetcaster/ui/player/PlayerScreen.kt

```diff
@@ -230,6 +249,83 @@ private fun PlayerContentTableTop(
     }
 }
 
+@Composable
+private fun PlayerContentBook(
+    uiState: PlayerUiState,
+    bookPosture: DevicePosture.BookPosture,
+    onBackPress: () -> Unit
+) {
+    val hingePosition = with(LocalDensity.current) { bookPosture.hingePosition.left.toDp() }
+    val hingeWidth = with(LocalDensity.current) { bookPosture.hingePosition.width().toDp() }
+
+    Row(modifier = Modifier.fillMaxSize()) {
+        // Content for the left part of the screen
```

Comment:
Nit: to make this composable more readable, would you mind extracting out the content on each part of the screen to a different composable? i.e.

```
Row {
    PlayerContentBookLeft(...)
    Spacer(hinge)
    PlayerContentBookRight(...)
}
```

WDYT?

Reply:
I like it.
I modified a bit the layout to keep the same `TopAppBar` layout.
The gradient is now also drawn over the hinge but the positioning of the active elements is done avoiding the possible empty space on dual-screen devices:
![Screenshot_20220317_151316](https://user-images.githubusercontent.com/188886/158826325-892355ed-3a8d-44e5-bf61-eda7ccfe5b69.png)